02-peer-protocol.md

```diff
@@ -856,6 +869,14 @@ seconds, and the protocol only supports an expiry in blocks.
 specification; larger amounts are not necessary, nor wise, during the
 bootstrap phase of the network.
 
+The node _responsible_ for paying the Bitcoin fee should maintain a "fee
+spike buffer" on top of its reserve to accommodate a future fee increase.
+Without this buffer, the node _responsible_ for paying the Bitcoin fee may
+reach a state where it is unable to receive any HTLC while maintaining its
+channel reserve (because of the increased weight of the commitment transaction),
+resulting in an unusable channel. See [#728](https://github.com/lightningnetwork/lightning-rfc/issues/728)
```

Comment:
Complete unusable, are better to say 'degraded', because dust htlcs are still possible?

Reply:
True, I'll update that, unusable is a bit strong :)